test/functional/rpc_getblockstats.py

```diff
@@ -160,6 +160,9 @@ def run_test(self):
         assert_raises_rpc_error(-1, 'getblockstats hash_or_height ( stats )', self.nodes[0].getblockstats, '00', 1, 2)
         assert_raises_rpc_error(-1, 'getblockstats hash_or_height ( stats )', self.nodes[0].getblockstats)
 
+        self.log.info('Test block height 0')
+        genesis_stats = self.nodes[0].getblockstats(0)
+        assert_equal(genesis_stats["subsidy"], 5000000000)
```

Comment:
```suggestion
        assert_equal(genesis_stats["blockhash"], "0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206")
```
I would suggest making the check more strict by checking for the exact expected blockhash, not just the subsidy which is the same in the next blocks ✌️

Reply:
done